### PR TITLE
win: remove unused code

### PR DESCRIPTION
**Pull request by saghul**
This code has been disabled since 2011, I guess we're ok without it.

### CODE REVIEW

**Comment by saghul** [comment]
maybe some other @libuv/collaborators ?

File: src/win/tcp.c [hunk 1]
```diff
@@ -488,24 +479,9 @@ static void uv_tcp_queue_read(uv_loop_t* loop, uv_tcp_t* handle) {
...
-    }
-    assert(handle->tcp.conn.read_buffer.base != NULL);
-    buf = handle->tcp.conn.read_buffer;
-  } else {
-    handle->flags |= UV_HANDLE_ZERO_READ;
-    buf.base = (char*) &uv_zero_;
-    buf.len = 0;
-  }
+  handle->flags |= UV_HANDLE_ZERO_READ;
+  buf.base = (char*) &uv_zero_;
```
Comment (bnoordhuis) [inline src/win/tcp.c]: Superfluous cast?  uv_zero_ is a mutable char array.

File: src/win/udp.c [hunk 2]
```diff
@@ -279,83 +274,35 @@ static void uv_udp_queue_recv(uv_loop_t* loop, uv_udp_t* handle) {
...
-      return;
-    }
-    assert(handle->recv_buffer.base != NULL);
+  handle->flags |= UV_HANDLE_ZERO_READ;
 
-    buf = handle->recv_buffer;
-    memset(&handle->recv_from, 0, sizeof handle->recv_from);
-    handle->recv_from_len = sizeof handle->recv_from;
-    flags = 0;
+  buf.base = (char*) uv_zero_;
```
Comment (bnoordhuis) [inline src/win/udp.c]: Ditto.

**Comment by bnoordhuis** [comment]
LGTM

File: src/win/tcp.c (hunk 1 above)
Comment (saghul) [inline src/win/tcp.c]: Actually, we only use it there, so I'm tempted to replace it with `""` since that memory won't be touched anyway. Thoughts?

**Comment by saghul** [comment]
Removed `uv_zero_` from tcp.c and udp.c. Also, CI: https://ci.nodejs.org/view/libuv/job/libuv-test-commit-windows/119/

**Comment by piscisaureus** [comment]
Very late +1

**Pull request merged**